### ISSUE TITLE
Simplify exception definitions

### ISSUE THREAD

**Issue by fmigneault** [crim-ca/weaver#215]
Many of the exception definitions are duplicated across `weaver.owsexceptions` and `weaver.exceptions`. 

`weaver.owsexceptions` is an old copy of twitcher code that is not specifically needed. 

`weaver.exceptions` could define by itself all the necessary XML/JSON content rendering (#125).
They should also preserve all handling logic to catch raised exceptions by corresponding `HTTPExceptions` (both `pyramid` ones from `weaver` app and `werkzeug` ones from `pywps` service - same name), to raise the expected HTTP status code, description, WPS locator, etc.
Finally, they should also handle the auto-conversion of `pywps.exceptions` classes. 

To summarize, make `weaver.exceptions` "all-in-one" class definitions to return expected responses no matter what caused them to raise.
